packages/engine/CONTRIBUTING.md

```diff
@@ -0,0 +1,149 @@
+[comment]: <> (This CONTRIBUTING guide was heavily inspired by the great one provided by the lovely folks at Atom [https://github.com/atom/atom/blob/master/CONTRIBUTING.md])
+
+# Contributing to hEngine
+
+Thank you for your interest in adding to the hEngine!
+
+We've provided the following guidelines to help you with contributions, in addition to the ones described below, we've
+also established a set of [community guidelines](https://hash.ai/legal/community) to enable as many people as possible
+to contribute to and benefit from HASH. Please follow these when interacting with this repo.
+
+#### Table Of Contents
+
+- [I have a question, where do I go?](#i-have-a-question--where-do-i-go-)
+
+- [Getting started, what do I need to know?](#getting-started--what-do-i-need-to-know-)
+    * [Design Decisions](#design-decisions)
+
+- [How Can I Contribute?](#how-can-i-contribute-)
+    * [Reporting Bugs](#reporting-bugs)
+    * [Suggesting Enhancements](#suggesting-enhancements)
+    * [Your First Code Contribution](#your-first-code-contribution)
+    * [Pull Requests](#pull-requests)
+
+- [Styleguides](#styleguides)
+    * [Rust Styleguide](#rust-styleguide)
+    * [JavaScript Styleguide](#javascript-styleguide)
+    * [Python Styleguide](#python-styleguide)
+    * [Documentation Styleguide](#documentation-styleguide)
+
+- [Additional Notes](#additional-notes)
+    * [Issue and Pull Request Labels](#issue-and-pull-request-labels)
+
+> **WIP** This document is a work in-progress, if the section you need isn't finished, please feel free to reach out to us for clarification!
+
+## I have a question, where do I go?
+
+> **Note:** Please don't use GitHub issues for help, we have a friendly community on...
+
+* [Discord](https://discord.com/invite/BPMrGAhjPh)
+* [Our Forum](https://community.hash.ai/)
+* And any other methods outlined on [our site](https://hash.ai/contact)
+
```

Comment:
This should either be marked as WIP or the `> **Note:** ` should be removed, otherwise the section starts with a quote-like formatting and is followed by a normally formatted list.

Reply:
Seems like it's a matter of stylistic choice. Atom do something similar:
![image](https://user-images.githubusercontent.com/25749103/148775592-833e0f15-74d1-4137-b493-3878f39cd3a3.png)

Depends on how you read it, if others raise concerns with it we can change it